projects/view/client/rtProject/rtProjectInfoSection.html

```diff
@@ -0,0 +1,45 @@
+<template name="rtProjectInfoSection">
```

Comment:
My personal opinion is that it is really much easier to read templating and html using tabs instead of spaces(have always used tabs for html and spaces for javascript, java and other programming languages)